tmva/pymva/test/EmitFromKeras.cxx

```diff
@@ -4,7 +4,7 @@
 //           The program is run when the target 'TestRModelParserKeras' is built.
 //           The program generates the required .hxx file after parsing a Keras .h5 file into a RModel object.
 
-
+#include "TMVA/RModel_Base.hxx"
```

Comment:
why this is needed  ? `RModel.hxx` should include already `RModel_Base.hxx`. 
Same for the EmitFromPyTorch file that is not existing anymore